rplugin/python3/deoplete/sources/deoplete_jedi/cache.py

```diff
@@ -237,7 +249,10 @@ def balanced(astr):
                 else:
                     open_d = ''
         return not stack, open_d
+
     return balanced
+
+
 balanced = _balanced()
```

Comment:
I don't like the extra newlines here (see https://github.com/PyCQA/pycodestyle/issues/400#issuecomment-260302015).